quantum/process_keycode/process_tap_term_keys.c

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2020 Vladislav Kucheriavykh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include "process_tap_term_keys.h"
+
+void tapping_term_report(void) {
+    char display[8];
+
+    snprintf(display, 8, "%d", tapping_term);
+
+    send_string((const char *)display);
+}
+
+bool process_tap_term_keys(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {
+        switch (keycode) {
+            case KC_TAPP:
+                tapping_term_report();
+                return true;
+
+            case KC_TAPU:
+                tapping_term += 5;
+                return true;
+
+            case KC_TAPD:
+                tapping_term -= 5;
```

Comment:
The change should be pulled out of the function here, added to their own functions, and exposed. 

Eg, it would be nice/neat/awesome to be able to change this using encoders, for instance.

Reply:
If `TAP_TERM_KEYS_ENABLE` is enabled, you get direct access to the `tapping_term` variable at a keymap-level, so you can easily do something like `tapping_term += 10` in encoder code and/or display the current global tapping term on an OLED screen.